docs/wiki/PolicyDeepDive.md

```diff
@@ -70,4 +70,171 @@ Please see the following wiki article that explains this process further: [Addin
+    | extend
+        assignmentScope = tostring(properties.scope),
+        assignmmentNotScopes = tostring(properties.notScopes),
+        assignmmentParameters = tostring(properties.parameters)
+    | project assignmentScope,
+        assignmmentNotScopes,
+        assignmmentParameters
+  ```
+
+- The above command will give a result similar to what is shown below
+
+  ![alz-custom-initiative-assignments](media/alz-update-initiative-with-builtin-03.png)
+
+- As can be seen this particular initiative is assigned with only a single parameter at the following levels in the management group structure
+  - Contoso/
+
+> Note
+that the provided example has a simple parameter set. If more complex parameters are assigned to a policy which is to be migrated those should be noted down. In that respect the possibility to download the query results as CSV could be leveraged.
+
+- Switch from Azure Resource Graph Explorer back to the Policy view
+- Change the scope to include the scope described above, and search for the relevant initiative
+
+  ![alz-delete-initiative-assignments](media/alz-update-initiative-with-builtin-04.png)
+
+- For each assignment, click the ellipsis and select Delete Assignment.
+- Once all initiative assignments are deleted, go to the Definitions pane, search for the initiative definition. Once found click the ellipsis and choose Delete Policy Definition
+
+    ![alz-custom-initiative-def-search](media/alz-update-initiative-with-builtin-01.png)
+
+- Follow the steps in [Manually synchronize your local repo with ALZ-Bicep](#manually-synchronize-your-local-repo-with-alz-bicep) to synchronize policy updates from the ALZ-Bicep repository to your local repo.
+
+### Manually synchronize your local repo with ALZ-Bicep
+
+To get the latest updates from the ALZ-Bicep repo and download to you local repo, do the following.
+- On a client where you have already cloned your version of the ALZ-Bicep repo, start a PowerShell command prompt
+- Navigate to the root of the local version of the repo
+- Verify that you are on the main branch and up-to-date by running the following commands
+
+  ```posh
+  git checkout main
+  git pull
+  ```
+
+- If you did not already, copy the [`Invoke-GitHubReleaseFetcher.ps1`](https://github.com/Azure/ALZ-Bicep/tree/main/.github/scripts/Invoke-GitHubReleaseFetcher.ps1) script into the following location in your ALZ-Bicep repository: `.github/scripts/`
+- Synchronize your local repo with the ALZ-Bicep repo by running the following
+command
+  ```posh
+  .github/scripts/Invoke-GitHubReleaseFetcher.ps1 -githubRepoUrl "https://github.com/Azure/ALZ-Bicep" -directoryAndFilesToKeep @("infra-as-code/bicep/modules/policy")
+  ```
+
+- When you have synchronized your repo perform the following verification tasks
```

Comment:
Have you tested this as by default the script will put it in a `releases/<version number>` folder? Do we need to add more guidance around how this can be utilised in this scenario?

Reply:
Agreed, updated guidance to cover this